docs/topics/about-rules.adoc

```diff
@@ -19,5 +19,5 @@ otherwise(action)
 {ProductShortName} provides a comprehensive set of standard migration rules out-of-the-box. Because applications may contain custom libraries or components, {ProductShortName} allows you to write your own rules to identify use of components or software that may not be covered by the existing ruleset.
 
 ifndef::rules-development-guide[]
```

Comment:
Why does this ifndef occur if the module is used ONLY in the RD guide?

Reply:
The module is also used in the Intro to MTA guide. I added the Intro's master.adoc to the list of assemblies that include thi module.